migrations/20200629223354_create_tx_out_address_index.js

```diff
@@ -0,0 +1,7 @@
+exports.up = async (knex) => knex.schema.table('tx_out', (table) => {
+  table.index('address', 'idx_tx_out_address', 'hash')
```

Comment:
In hindsight, I would prefix the indices with "adalite" to prevent them from clashing with future indices added by cardano-db-sync migrations

Reply:
This will be solved by moving migrations to cardano-db-sync which arose due to potential deadlocks in migrations.